manager/api/export-import/src/main/java/io/apiman/manager/api/exportimport/manager/StorageExporter.java

```diff
@@ -39,19 +40,18 @@
 import io.apiman.manager.api.config.Version;
 import io.apiman.manager.api.core.IStorage;
 import io.apiman.manager.api.core.exceptions.StorageException;
-import io.apiman.manager.api.core.logging.ApimanLogger;
 import io.apiman.manager.api.exportimport.i18n.Messages;
 import io.apiman.manager.api.exportimport.write.IExportWriter;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
 import java.util.Date;
 import java.util.Iterator;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
 
 @ApplicationScoped
 public class StorageExporter {
-    @Inject @ApimanLogger(StorageExporter.class)
-    private IApimanLogger logger;
+    private final IApimanLogger logger = ApimanLoggerFactory.getLogger(StorageExporter.class);
```

Comment:
Any reason to not make this also static final?

Reply:
See previous